final/src/main/java/ua/epam/spring/hometask/controller/AuditoriumController.java

```diff
@@ -1,28 +1,32 @@
 package ua.epam.spring.hometask.controller;
 
+import org.hibernate.validator.constraints.NotBlank;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
+import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.servlet.ModelAndView;
 import ua.epam.spring.hometask.facade.AuditoriumFacade;
 
 @Controller
 @RequestMapping("/auditoriums")
+@Validated
 public class AuditoriumController {
 
     @Autowired
     private AuditoriumFacade auditoriumFacade;
 
-    @GetMapping
-    public String viewAll(Model model) {
+    @RequestMapping(method = RequestMethod.GET)
+    public String index(Model model) {
```

Comment:
I highly recommend to follow code conventions and use verbs as the first word in a method name.